app/main.py

```diff
@@ -1 +1,10 @@
-# write your code here
+import os
+
+
+def move_file(command: str) -> None:
+    cmd, file_name, direction = command.split()
+    dir_ = os.path.dirname(direction)
+
+    if dir_:
+        os.makedirs(dir_, exist_ok=True)
```

Comment:
You could use here operator warlus operator 

```
if dir_ := os.path.dirname(direction):
    os.makedirs(dir_, exist_ok=True)
```

That way you code will be a bit cleaner : )